site/content/en/guides/example/multi_base.md

```diff
@@ -173,4 +173,6 @@ spec:
 ```
 
 Similarly to adding different `namePrefix` in different variants, one can also add different `namespace` and compose those variants in
-one _kustomization_. For more details, take a look at [multi-namespaces](multi-namespace.md).
+one _kustomization_. For more details, take a look at:
+- [Setting Namespaces](../config_management/namespaces_names.md)
```

Comment:
This link doesn't ascend far enough and is broken. Please check the deploy preview created by the CI checks.

Reply:
Had you checked it in preview ? It's working, I think you saw old CI checks.

Before I was looking for the multi-namespaces file, let me know if you have the link.

Reply:
The link original was broken but this one is similar for explanation